Python/pythonrun.c

```diff
@@ -251,17 +251,21 @@ PyRun_InteractiveOneObjectEx(FILE *fp, PyObject *filename,
         }
         return -1;
     }
-    m = PyImport_AddModuleObject(&_Py_ID(__main__));
-    if (m == NULL) {
+
+    PyObject *module = PyImport_AddModuleRef("main");
```

Comment:
```suggestion
    PyObject *module = PyImport_AddModuleRef("__main__");
```

Reply:
Well spotted, I found the same typo. I wrote commit https://github.com/python/cpython/commit/a5c2ad0c3d23d2b1e61ab8e0d7ee64f7e1288547 to refactor the code in a separated commit. Moreover, I left this code unchanged for now, since PyImport_AddModuleObject() is no longer deprecated by this PR. Also, with that change, the  ``"__main__"`` UTF-8 string has to be decoded at each function call.